lib/task_bunny/worker.ex

```diff
@@ -183,6 +168,11 @@ defmodule TaskBunny.Worker do
     {:reply, status, state}
   end
 
+  @spec pname(job :: atom) :: atom
+  defp pname(job) do
+    String.to_atom("TaskBunny.Worker.#{job}")
```

Comment:
I just moved the function as I felt it's good idea to have signal handling same places.